src/diffusers/loaders/lora.py

```diff
@@ -303,7 +311,7 @@ def _best_guess_weight_name(cls, pretrained_model_name_or_path_or_dict, file_ext
             targeted_files = [
                 f for f in os.listdir(pretrained_model_name_or_path_or_dict) if f.endswith(file_extension)
             ]
-        else:
+        elif not local_files_only:
```

Comment:
I don't understand why this change. Given we raise an exception above if `local_files_only`, then it's not needed to check it here, right?

Reply:
Right. Resolved in 88b93c92.